HW22/roles/hw22/tasks/hosts_file.yml

```diff
@@ -0,0 +1,23 @@
+- name: Find out host ip
+  ansible.builtin.set_fact: host_ip={{ ansible_ssh_host }}
```

Comment:
Which another option you have to get host ip?

Reply:
host_ip=hostvars[inventory_hostname]['ansible_env'].SSH_CONNECTION.split(' ')[2]